src/desktop/apps/gallery_partnerships/client/index.coffee

```diff
@@ -8,6 +8,11 @@ module.exports.init = ->
     $('.gallery-partnerships2__chat__video').hide()
     $('.gallery-partnerships2__chat__fallback_image').show()
 
+  isUserAdmin = sd.CURRENT_USER && sd.CURRENT_USER.type == 'Admin'
+
+  if sd.INTERCOM_ENABLED && sd.INTERCOM_APP_ID && !isUserAdmin
+    intercom = require('./intercom')
```

Comment:
Looks like we had to initialize the Intercom script in js vs. inserting the snippet right before the `<body>` ends. Not sure if we have to wait for the DOM to be ready to avoid some race conditions. Let's give it a try!